### DISCUSSION TITLE
建议同一个资源组可以配置多个流程

### DISCUSSION THREAD

**Discussion by anchen418857063** [hhyo/Archery#1688] [Ideas]
### 目前遇到的问题/使用障碍
公司日常业务修改数据dml 特别多，DBA操作操作工单的工作非常繁重，并且经常有深夜需要协助执行工单的情况
### 希望如何解决/实现它
建议同一资源组可以配置多个流程按数据库操作类型来区分：提交工单界面新增下拉选择流程downlist，由发起人选择需要走哪个流程。这样我的dml 操作可以配置一个流程，审批流程里面就到开发leader就可以执行，不需要到DBA。而只要包含ddl操作的工单必须选择到DBA才能执行。

**Comment by nick2wang** [comment]
由发起人选择不可靠且繁琐，倒不如把审批流程细分为ddl和dml分开配置

**Reply by boldmanQ** [reply]
这个已经有开发计划了吗？

**Comment by anchen418857063** [comment]
这个也行

**Comment by peng19832** [comment]
dml，dba我觉着没必要审核，因为除了项目内专职的那种开发dba，像运维开发分开的场景，运维dba根本不清楚dml的正确性，所以，直接配置dml，研发自己审核执行就行了，archery负责备份sql，提供回滚就行
